pull-request.sh

```diff
@@ -71,10 +73,22 @@ create_pull_request() {
     # Option 2: Open a new pull request
     else
         # Post the pull request
-        DATA="{\"title\":${TITLE}, \"body\":${BODY}, \"base\":${TARGET}, \"head\":${SOURCE}, \"draft\":${DRAFT}}"
+        DATA="{\"title\":${TITLE}, \"body\":${BODY}, \"base\":${TARGET}, \"head\":${SOURCE}, \"draft\":${DRAFT}, \"maintainer_can_modify\":${MAINTAINER_CAN_MODIFY}}"
         echo "curl --user ${GITHUB_ACTOR} -X POST --data ${DATA} ${PULLS_URL}"
-        curl -sSL -H "${AUTH_HEADER}" -H "${HEADER}" --user "${GITHUB_ACTOR}" -X POST --data "${DATA}" ${PULLS_URL}
-        echo $?
+        PR_RES=$(curl -sSL -H "${AUTH_HEADER}" -H "${HEADER}" --user "${GITHUB_ACTOR}" -X POST --data "${DATA}" ${PULLS_URL})
+        echo "PR open request result code: $?"
+        echo ${PR_RES}
+        PR_NUMBER=$(echo ${PR_RES} | jq --raw-output '.number')
+
+        echo "Created PR NUMBER: $PR_NUMBER"
+        if [ -n "$PULL_REQUEST_ASSIGNEE" ]; then
```

Comment:
This is passed to the calling function but not parsed in the beginning. Please after you update these fixes to use the action on the repository to demonstrate the following cases:

 - regular pull request works
 - pull request with maintainer can't modify flag is set
 - pull request with assignee works as expected.

Thanks!